data/json/flags.json

```
@@ -616,7 +616,7 @@
   {
     "id": "PALS_LARGE",
     "type": "json_flag",
-    "info": "This item will <good>attach directly</good> to <info>load bearing vests</info> take a <bad>large amount of space</bad>."
+    "info": "This item will <good>attach directly</good> to <info>load bearing vests</info> taking a <bad>large amount of space</bad>."
```

```suggestion
    "info": "This item will <good>attach directly</good> to <info>load bearing vests</info>, taking a <bad>large amount of space</bad>."
```

Fairly sure this needs a comma, but I am not a native english speaker.